doc/code/qml_tape.rst

```diff
@@ -0,0 +1,160 @@
+qml.tape
+========
+
+.. warning::
+
+    The new PennyLane tape mode is **experimental**, and does not currently have feature-parity with
+    the existing QNode. `Feedback and bug reports
+    <https://github.com/PennyLaneAI/pennylane/issues>`__ are encouraged and will help improve the
+    new tape mode.
+
+    Tape mode can be enabled globally via the :func:`~.enable_tape` function, without changing your
+    PennyLane code:
+
+    >>> qml.enable_tape()
+
+    Once enabled, tape mode can be disabled via :func:`~.disable_tape`.
+
+
+Tape-mode QNodes
+----------------
+
+The PennyLane tape module provides a new QNode class, rewritten from the ground-up,
+that uses a :class:`~.QuantumTape` to represent the internal variational quantum circuit.
+Tape mode provides several advantanges over the standard PennyLane QNode.
+
+* **No more Variable wrapping**: In tape mode, QNode arguments no longer become :class:`~.Variable`
+  objects within the QNode; arguments can be easily inspected, printed, and manipulated.

```

Comment:
Just a thought here: this describes a (major) change that having a `QuantumTape` brought. Would it be worth including it in the changelog/including it here but after some time rewording it? This would be just important, as at one point when tapes have feature parity and have been used for some time, `Variable` would be fully deprecated and (referring to it may lead to confusion)

Reply:
Yep! This page is just a placeholder of sorts, to keep track over what tape mode allows, and all the changes. We can incorporate these examples and add them to the changelog prior to release.